src/set_theory/game/pgame.lean

```diff
@@ -249,6 +249,29 @@ instance : has_one pgame := ⟨⟨punit, pempty, λ _, 0, pempty.elim⟩⟩
 instance unique_one_left_moves : unique (left_moves 1) := punit.unique
 instance is_empty_one_right_moves : is_empty (right_moves 1) := pempty.is_empty
 
+/-- The game `{0 | x}`. When `x` is `numeric` and positive, this yields a positive `numeric`
+pre-game strictly lesser than `x`. -/
+def of_right_move_with_zero (x : pgame) : pgame := ⟨punit, punit, 0, λ _, x⟩
```

Comment:
Instead of having this awkwardly named def, should we have notation for `{x | y}` indexed by `punit`?

Reply:
That would be really nice, and would allow us to simplify a few definitions. Might also be nice to have notation `{x | }` and `{ | x}` with the obvious indexing types.

How would we refer to this construction on the lemma names, though?

Reply:
I would guess something like
```
@[reducible] def of (x y : pgame) : pgame := ⟨punit, punit, λ _, x, λ _, y⟩
@[reducible] def of_left (x : pgame) : pgame := ⟨punit, pempty, λ _, x, pempty.elim⟩
@[reducible] def of_right (x : pgame) : pgame := ⟨pempty, punit, pempty.elim, λ _, x⟩
```
and then attach notation to those definitions.

Reply:
I'll put this in another PR, and then add the numeric theorems on this PR.

Reply:
This notation seems to work out alright:
```
notation `P{` x ` | ` y `}` := of x y
notation `P{` x ` |}` := of_left x
notation `P{| ` y `}` := of_right y
```

Reply:
maybe call them single_single, single_none or single_empty, none_single or empty_single